src/WebJobs.Script/ExtensionBundle/ExtensionBundleManager.cs

```diff
@@ -23,16 +25,18 @@ public class ExtensionBundleManager : IExtensionBundleManager
     {
         private readonly IEnvironment _environment;
         private readonly ExtensionBundleOptions _options;
+        private readonly FunctionsHostingConfigOptions _configOption;
         private readonly ILogger _logger;
         private readonly string _cdnUri;
         private string _extensionBundleVersion;
 
-        public ExtensionBundleManager(ExtensionBundleOptions options, IEnvironment environment, ILoggerFactory loggerFactory)
+        public ExtensionBundleManager(ExtensionBundleOptions options, IEnvironment environment, ILoggerFactory loggerFactory, FunctionsHostingConfigOptions configOption)
```

Comment:
An alternative approach would also be to add the values you need directly to `ExtensionBundleOptions` and update its configuration helper to pull out the two features we need.

This is more in line with that `IOptions` pattern is intended for. Where the options object you consume is scenario-specific and the construction of it is separated from the usage.